litmus-portal/frontend/src/views/ChaosWorkflows/BrowseAnalytics/WorkflowComparisonTable/TableToolbar.tsx

```diff
@@ -0,0 +1,251 @@
+import {
+  Typography,
+  InputBase,
+  InputAdornment,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  Button,
```

Comment:
If it is special use case for not using custom button then ignore otherwise PTAL @ishangupta-ds

Reply:
same